app/views/users/_user_card.html.erb

```diff
@@ -0,0 +1,7 @@
+<article class="user_card">
+    <img src="https://pm1.narvii.com/7258/5520799cf0539b408bd8abee0a14d3a492ee5107r1-753-753v2_hq.jpg" alt="User photo">
+    <div class="user_info">
+        <h2><%= user.name %></h2>
+        <h4>Number of post: <%= user.posts_counter %></h4>
+    </div>
+</article>
```

Comment:
- [ ] Good job so far 👏 It looks like you are trying to render a static image for each user while every user should have a unique image. Please add a user link in the image tag and it should probably look like this `<img src=<%= user.photo%> alt="User photo">`